SPEC.md

```diff
@@ -4,7 +4,7 @@ Inspired by [TomDoc](http://tomdoc.org), KSS attempts to provide a methodology f
 
 ## Purpose
 
-KSS is a set of guidelines to help you produce an HTML styleguide tied to CSS documentation that is nice to read in plain text, yet structured enough to be automatically extracted and processed by a machine. It is designed with CSS preprocessors (such as SCSS or LESS) in mind, and flexible enough to accommodate a multitude of CSS frameworks (such as YUI, Blueprint or 960).
+KSS is a set of guidelines to help you produce an HTML styleguide tied to CSS documentation that is nice to read in plain text, yet structured enough to be automatically extracted and processed by a machine. It is designed with CSS preprocessors (such as Sass or LESS) in mind, and flexible enough to accommodate a multitude of CSS frameworks (such as Bootstrap or Foundation).
```

Comment:
The preprocessor is called "Sass", not "SCSS".

Young-uns don't even know what YUI is. Using popular 2015-era CSS frameworks. We'll have to revisit this in another 5 years, I'm sure. :-)